src/azurelinux/3.0/net9.0/crossdeps-builder/amd64/Dockerfile

```diff
@@ -4,6 +4,7 @@ RUN tdnf install -y \
         # Common utilities
         ca-certificates \
         git \
+        pigz \
```

Comment:
We don't need it in the crossdeps-builder images as we only use these images to build components that are included in other images.

Reply:
```suggestion
```